src/app/components/Question.js

```diff
@@ -0,0 +1,87 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+class Question {
+  constructor(typeOfQuestion, howManyAnswers) {
+    this._typeOfQuestion = typeOfQuestion;
+    this._howManyAnswers = howManyAnswers;
+    this._rightAnswer = this._createRandomInt(this._howManyAnswers);
+    this._answers = [];
+    this._questionData = {};
+  }
+
+  get questionData() {
+    return this._questionData;
+  }
+
+  _getMaxId() {
```

Comment:
Nie powinniśmy używać switch Case ze względu na wydajnosc i jakosc  kodu, tu jest ciekawy artykuł https://dev.to/lukyhenson/replace-your-switch-statement-and-multiple-if-and-else-using-object-literals-en-us-1dec. W tym projekcie na razie tak zostawmy bo nie skupiamy sie jeszcze na wydajności, ale tak na przyszłość